### PR TITLE
[Term Entry] D3 Selection .append() added

### LINKED ISSUE #3958: [Term Entry] D3 Selection .append()
### Reason/inspiration (optional)

We would like a new term entry on the `.append()` method for [D3](https://www.codecademy.com/resources/docs/d3): Selection. The entry should go in a new file under `docs/content/d3/concepts/selection/terms/append/append.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by antriksh-9**
### Description

I was assigned this [issue](https://github.com/Codecademy/docs/issues/3958).
Created Codecademy Docs term entry on the .append() method for D3 Selection.
Closes #3958.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/d3/concepts/selection/terms/append/append.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - 'introduction-to-javascript'
+  - 'paths/computer-science'
+---
+
+The **`selection.append()`** function is used to append a new element to the `HTML` tag name as given in the parameters to the end of the element. If the type that is given is a function then it must be evaluated for each element that is in the selection.
+
+## Syntax
+
+```pseudo
+selection.append(name);
```
Comment (avdhoottt) [inline content/d3/concepts/selection/terms/append/append.md:22]:
```suggestion
selection.append(type);

```
The `type` word would describe the parameter correctly.

File: content/d3/concepts/selection/terms/append/append.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+- `name`: This parameter takes a string that defines the type of the element.
```
Comment (avdhoottt) [inline content/d3/concepts/selection/terms/append/append.md:24]:
```suggestion
- `type`: This parameter takes a string that defines the type of the element.

```

File: content/d3/concepts/selection/terms/append/append.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  - 'Methods'
+  - 'Functions'
+  - 'Elements'
+  - 'Selection'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/computer-science'
+---
+
+The **`selection.append()`** function is used to append a new element to the `HTML` tag name as given in the parameters to the end of the element. If the type that is given is a function then it must be evaluated for each element that is in the selection.
```
Comment (avdhoottt) [inline content/d3/concepts/selection/terms/append/append.md:17]:
```suggestion
The **`.append()`** function is used to append a new element to the `HTML` tag name as given in the parameters to the end of the element. If the type that is given is a function then it must be evaluated for each element that is in the selection.

```
This line contains some plagiarism, please change it.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@antriksh-9 I might have some more changes in the future. For now, make the mentioned changes.

File: content/d3/concepts/selection/terms/append/append.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  bodySelection.append('p')
+    .text('This is a new paragraph added using D3.js');
+</script>
+
+</body>
+</html>
+```
+
+In this example, D3.js selects the `<body>` element and appends a new `<p>` (paragraph) element to it. The `.text()` method is then used to set the text content 
+of the newly appended paragraph.
```
Comment (avdhoottt) [inline content/d3/concepts/selection/terms/append/append.md:55]:
```suggestion
In this example, D3.js selects the `<body>` element and appends a new `<p>` (paragraph) element to it. The `.text()` method is then used to set the text content of the newly appended paragraph.

```

**Review by avdhoottt — APPROVED** [review]
Looks good for the second review.

**Review by dakshdeepHERE — APPROVED** [review]
Did some minor changes and this LGTM 🚀

**Pull request merged**